### PR TITLE
dw-2816 - switch to using alpine over jdk-slim

### PR DESCRIPTION
**Pull request by benj-fletch**

### CODE REVIEW

File: Dockerfile_integration [hunk 1]
```diff
@@ -15,7 +15,9 @@ COPY settings.gradle.kts .
 COPY gradle.properties .
 RUN $GRADLE build && $GRADLE test
 
-RUN apt-get update && apt-get install -y gosu && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add su-exec \
+    && rm -rf /var/lib/apt/lists/*
```
Comment (mattburgess) [inline Dockerfile_integration]: That last line can go now that we're not using `apt`

File: docker/hbase/Dockerfile [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 ENV HBASE_MANAGES_ZK=false
 
 WORKDIR /
 
-RUN apt-get update && \
-    apt-get install -y procps wget && \
+RUN apk update && \
+    apk add --no-cache procps wget tar bash && \
     rm -rf /var/lib/apt/lists/*
```
Comment (mattburgess) [inline docker/hbase/Dockerfile]: Likewise

File: entrypoint.sh [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+#!/bin/sh
 
 set -e
 
 # Create a user to run the process as instead of root
 
-: ${SUID:=1000}
-: ${SGID:=1000}
+: "${USER_NAME:=user}"
+: "${GROUP_NAME:=usergroup}"
```
Comment (mattburgess) [inline entrypoint.sh]: What do `user` and `usergroup` expand to here, and how? Should they not be env vars?

File: Dockerfile_integration (hunk 1 above)
Comment (benj-fletch) [reply Dockerfile_integration]: Good catch, will change

File: docker/hbase/Dockerfile (hunk 2 above)
Comment (benj-fletch) [reply docker/hbase/Dockerfile]: will change

File: entrypoint.sh (hunk 3 above)
Comment (benj-fletch) [reply entrypoint.sh]: I don't think they expand to anything to be honest, I was following the previous syntax. I am happy to change these to env vars in the Dockerfile.

**Pull request merged**